saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 
 set -ex
+
+ACC_KEY_TMP_FILE=$(mktemp)
+chmod 600 "$ACC_KEY_TMP_FILE"
+trap 'rm -f "$ACC_KEY_TMP_FILE"' EXIT
```

Comment:
I understand why you added this, but there is no need for it as long as the file is located in the LUKS volume (e.g. when using a location under `/etc/ccmv2/`).